### DISCUSSION TITLE
Movies in watchlist are not being scraped for torrents

### DISCUSSION THREAD

**Discussion by echotx** [itsToggle/plex_debrid#591] [Q&A]
Hello Everyone,

As far as I can tell, I have everything set up correctly per the installation instructions. My r clone drive is working and when I add a movie to my watchlist plex_debrid recognizes the movie being there but nothing further happens. Am I missing a step somewhere?

**Comment by thomaslc66** [comment]
I have the same issue but only for some movies.
As I've seen it seems that there's some rules for the scrapping and if nothing match the torrent is not downloaded.
You can edit the rules but you need to understand which ones are not match (what I'm currently doing)
